pkg/gui/arrangement.go

```diff
@@ -337,10 +337,12 @@ func (gui *Gui) currentSideWindowName() string {
 
 	for idx := range gui.State.ContextManager.ContextStack {
 		reversedIdx := len(gui.State.ContextManager.ContextStack) - 1 - idx
-		context := gui.State.ContextManager.ContextStack[reversedIdx]
+		ctx := gui.State.ContextManager.ContextStack[reversedIdx]
 
-		if context.GetKind() == types.SIDE_CONTEXT {
-			return context.GetWindowName()
+		if ctx.GetKind() == types.SIDE_CONTEXT {
+			return ctx.GetWindowName()
+		} else if ctx.GetKey() == context.PATCH_BUILDING_MAIN_CONTEXT_KEY {
+			return "commits"
```

Comment:
I have no idea if this is an appropriate way to fix it, to be honest. It seems to work for me. 😄

Reply:
I'm surprised the original code isn't working: there should be some side context in the context stack, and I'd expect to see the a context in there belonging to the commits window, given you had to go through the commits window to reach the patch building window. I'd like to find the root cause of the issue before we add something this specific to this file

Reply:
Ok, I dropped this and pushed a new commit that looks more like the appropriate fix to me. Please have a look.

I still don't have enough overview over how things play together to tell whether this might have broken something else.

Reply:
We've got enough integration tests for these things that I'm pretty confident we haven't broken anything, and if we stumble across a problem we can add a test for it and revisit